src/jsdoc_transformer.ts

```diff
@@ -710,24 +710,26 @@ export function jsdocTransformer(
       }
 
       function visitImportDeclaration(importDecl: ts.ImportDeclaration) {
-        // No need to forward declare side effect imports.
+        // For each import, insert a goog.requireType for the module, so that if TypeScript does not
+        // emit the module because it's only used in type positions, the JSDoc comments still
+        // reference a valid Closure level symbol.
+
+        // No need to requireType side effect imports.
         if (!importDecl.importClause) return importDecl;
-        // Introduce a goog.forwardDeclare for the module, so that if TypeScript does not emit the
-        // module because it's only used in type positions, the JSDoc comments still reference a
-        // valid Closure level symbol.
+
         const sym = typeChecker.getSymbolAtLocation(importDecl.moduleSpecifier);
         // Scripts do not have a symbol, and neither do unused modules. Scripts can still be
         // imported, either as side effect imports or with an empty import set ("{}"). TypeScript
         // does not emit a runtime load for an import with an empty list of symbols, but the import
         // forces any global declarations from the library to be visible, which is what users use
-        // this for. No symbols from the script need forward declaration, so just return.
+        // this for. No symbols from the script need requireType, so just return.
```

Comment:
we might want to revisit this part in the future. If users import code just for the global symbols it declares, emitting a `goog.requireType` that doesn't assign to a local variable might be the right thing to do; except I don't know if that's supported for `goog.requireType`.

Can you leave a TODO?